images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -286,6 +286,6 @@ function Get-PipxVersion {
 }
 
 function Get-GraalVMVersion {
-    $version = $GRAALVM_BIN/java --version | Out-String | Select-Object -First 2 | Take-OutputPart -Part 7
+    $version = & "$env:GRAALVM_11_BIN\java" --version | Out-String | Select-Object -First 2 | Take-OutputPart -Part 7
```

Comment:
I think we also should print environment variable to docs to notify customers where we install GRAALVM

Reply:
Added table contains `Version` and `Environment variables` columns for GraalVM